packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/About.1.tool.mdx

```diff
@@ -208,7 +358,11 @@ it('should validate the screenshot with options', () => {
 
 ### Terra.validates.element
 
-The element assertion basically performs both `Terra.validates.accessibility` and `Terra.validates.screenshot` assertions. Like these two assertions the element assertion should also only be invoked inside an `it` block.
+The validates element assertion performs both accessibility and screenshot assertions.
+
+<Notice variant="caution" ariaLevel="3">
+  The validates element assertion must be used within an `it` block.
+</Notice>
 
 ```js
 it('should validate the element', () => {
```

Comment:
Could we add a comment on what the default mismatch tolerance is for validate element and screenshot?

Reply:
I added the default tolerance to the option descriptions:

https://github.com/cerner/terra-toolkit/pull/610/commits/36606851682a8a89e5cc76e7a6ec39436bd45d0c